Directory.Packages.props

```diff
@@ -74,37 +74,37 @@
     <PackageVersion Include="Dapper" Version="2.1.28" />
     <PackageVersion Include="Dapr.AspNetCore" Version="1.12.0" />
     <PackageVersion Include="DnsClient" Version="1.7.0" />
-    <PackageVersion Include="Grpc.AspNetCore" Version="2.59.0" />
-    <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.59.0" />
-    <PackageVersion Include="Grpc.Tools" Version="2.59.0" />
+    <PackageVersion Include="Grpc.AspNetCore" Version="2.60.0" />
+    <PackageVersion Include="Grpc.Net.ClientFactory" Version="2.60.0" />
+    <PackageVersion Include="Grpc.Tools" Version="2.60.0" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="KubernetesClient" Version="13.0.11" />
-    <PackageVersion Include="JsonSchema.Net" Version="5.4.0" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.2.0" />
-    <PackageVersion Include="Microsoft.FluentUI.AspNetCore.Components.Icons" Version="4.2.0" />
-    <PackageVersion Include="MongoDB.Driver" Version="2.22.0" />
+    <PackageVersion Include="JsonSchema.Net" Version="5.5.0" />
```

Comment:
it looks like there is a 6 version out. does the tool only do minor version updates?

Reply:
The tool is configurable, you can version lock for any version, Same major, or same Minor. If this is still outdated, it is because we don't have the newer versions in our feeds. The tool will only look at the feeds in the NuGet.config file.